Unable to load primitives with 2.1.2

**Describe the bug**

After upgrading from 2.1.1 to 2.1.2 I can no longer load the library via Node (I'm using Next.js). When the code runs I get this build error:

```
error - Error: Cannot find module '/home/myapp/node_modules/@react-pdf/primitives/lib/index.js'. Please verify that the package.json has a valid "main" entry
```

Downgrading to 2.1.1 fixes the issue.

**To Reproduce**
1. Install `@react-pdf/renderer` version `2.1.2` into any Next.js boilerplate app
2. Create a new `/api/...` route
3. Add this code to the API route: `import {Document, Page, StyleSheet, Text, View} from '@react-pdf/renderer';`
4. Run the code

**Expected behavior**
No failure

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Node
 - React-pdf version: 2.1.2

--- a/.changeset/selfish-planets-arrive.md
+++ b/.changeset/selfish-planets-arrive.md
@@ -1,0 +1,5 @@
+---
+'@react-pdf/primitives': patch
+---
+
+publish commonjs files to npm

--- a/packages/primitives/package.json
+++ b/packages/primitives/package.json
@@ -17,6 +17,7 @@
     "test": "jest"
   },
   "files": [
+    "lib",
     "src"
   ]
 }

